<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schedule</class>
 <widget class="QDialog" name="Schedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>316</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item alignment="Qt::AlignLeft">
     <widget class="QSlider" name="hSlider_1">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_1">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_persent_1">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_action1">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Nothing</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>316</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item alignment="Qt::AlignLeft">
     <widget class="QSlider" name="hSlider_3">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_3">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_persent_3">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_action3">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Nothing</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>316</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item alignment="Qt::AlignLeft">
     <widget class="QSlider" name="hSlider_2">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_2">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_persent_2">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_action2">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Nothing</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>316</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item alignment="Qt::AlignLeft">
     <widget class="QSlider" name="hSlider_4">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_4">
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_persent_4">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_action4">
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans T Chinese Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Nothing</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_action">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans T Chinese Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Action to do</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>label_action</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
